packages/core/src/components/navbar/navbarDivider.tsx

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the terms of the LICENSE file distributed with this project.
+ */
+
+import * as classNames from "classnames";
+import * as React from "react";
+import * as Classes from "../../common/classes";
+import { IProps } from "../../common/props";
+
+// allow the empty interface so we can label it clearly in the docs
+// tslint:disable-next-line:no-empty-interface
```

Comment:
in my experience, putting a comment inside makes it non-empty so you don't have to disable the rule. is this still the case?

Reply:
Nope, not the case anymore